docs/postgres-docker-db

```diff
@@ -0,0 +1,52 @@
+# Postgres
+
+
+## Postgres in docker
+
+To start you must first pull the official postgres docker image, this will
+be used to create and run postgres containers.
+
+`docker pull postgres`
+
+This pulls the official postgres docker image from the repo. Once you have the image
+on your machine you don't have to run this again. To check if you already have
+this image, use 'docker images' and check if postgres is listed.
+
+
+Once you have the postgres image, run the following command:
+
+`docker run --name postgres -e POSTGRES_PASSWORD=Password -p 5432:5432 -d postgres`
+
+This will create a container running postgres at port 5432. After this you can always
+start the above container with:
+
+`docker start postgres`
+
+
+## Docker compose:
+
+To deploy our database to AWS or builds that are not running a local container,
+run the command
```

Comment:
I would add in some instruction here about making sure the postgres/vanilla individual docker containers are stopped if you have them going, as there will be port issues if they are running and you try to run docker-compose up.  Also probably add detail about making sure the name of the vanilla image matches whats expected in the docker-compose.yml.  I added a bunch of docker docs and such in the Queue Consumer branch if you just want to grab that.  Just realized that branch is really old now, but there a bunch of docker work in there that would step all over this if we ever do want to merge it. 

https://github.com/cawasser/vanilla/pull/40/files